readthedocs/projects/models.py

```diff
@@ -208,9 +208,13 @@ class Project(models.Model):
         choices=constants.DOCUMENTATION_CHOICES,
         default='sphinx',
         help_text=_(
-            'Type of documentation you are building. <a href="'
-            'http://www.sphinx-doc.org/en/stable/builders.html#sphinx.builders.html.'
-            'DirectoryHTMLBuilder">More info on sphinx builders</a>.',
+            "Specify the type of documentation you are building. "
+            "This will make the automatic Read the Docs builder detect your MkDocs or Sphinx "
+            "configuration and build your project running Read the Docs' default command. "
+            "Using a readthedocs.yaml "
+            '<a href="https://docs.readthedocs.io/en/stable/config-file/index.html">configuration '
+            "file</a> allows you to change defaults or build documentation for <strong>other "
+            "tools than Sphinx and MkDocs</strong>."
```

Comment:
This is much better! 💯 

Pinging @agjohnson since he has probably thought about how to present this in the new templates, and may be good his input here.

Pinging @ericholscher because I assume he would like to explicitly mention other doctools here instead of just saying "other tools than Sphinx and MkDocs" 😉

Reply:
Yea, I think this is a small change, when we already have a big warning at the top of the page? This help text feels overwhelming, I think it should be made much smaller and just link to more information.

Reply:
> when we already have a big warning at the top of the page?

That warning doesn't say anything about supporting other tools than Sphinx. So if you look at the page with "normal user's perception", you'll see that RTD looks like a Sphinx-only platform (with a warning about using a configuration file).

I'll be jumping over to the Dashboard v2 work soon, where we can improve the combined UI... just wanna be sure that we push in some of the "good news" so that v1 users (i.e. all current users) aren't kept in the dark about support for other builders. Recall that the old help_text of this field unfortunately makes it look like we only have Sphinx builders.

I think we have a lot of "habbit users" that won't read our newsletters, tweets etc. but will discover information in odd places like this. So I don't really mind overloading the help_text for that reason, it's worth a shot.. in any case, we get rid of the "Sphinx builders" misunderstanding :)

Reply:
It'd be cool if Django model fields had a `help_text` AND a `doc_url` property or some other super-cool way to jump from a form into a how-to or a reference :)

But yeah, if we can just find a nice balance of writing brief help_texts that generally link to documentation, that will definitely benefit users and also help to showcase how an otherwise external documentation website can be made accessible on an application website.